services/core-api/app/api/email_preview/resources/email_preview_resource.py

```diff
@@ -0,0 +1,230 @@
+"""
+Email Preview Resource
+Provides development endpoints for previewing email templates with test data
+"""
+
+from flask import current_app, request, jsonify, Response
+from flask_restx import Resource, reqparse
+
+from app.extensions import api
+from app.api.utils.access_decorators import requires_any_of
+from app.api.utils.resources_mixins import UserMixin
+from app.api.utils.access_decorators import VIEW_ALL, MINESPACE_PROPONENT
+from app.api.constants import CORE_PURPLE_LOGO_BASE64_ENCODED, MINESPACE_LOGO_BASE64_ENCODED, BC_GOV_LOGO_BASE64_ENCODED
+
+
+class EmailPreviewResource(Resource, UserMixin):
+    """
+    Email Preview API endpoint for development
+    """
+    
+    @requires_any_of([VIEW_ALL, MINESPACE_PROPONENT])
```

Comment:
Would suggest locking this down to just core users (maybe the `core_admin` role)